gstd/src/msg/async.rs

```diff
@@ -82,15 +92,15 @@ impl Future for MessageFuture {
             },
         }
     }
-}
+} 
 
 impl FusedFuture for MessageFuture {
     fn is_terminated(&self) -> bool {
         !signals().waits_for(self.waiting_reply_to)
     }
 }
 
-/// Send a message and wait for reply.
+/// Send a message and wait for reply (as encoded structure).
```

Comment:
Need to somehow say that input should be codec Encodable, while output is codec Decodable